package.json

```diff
@@ -9,14 +9,15 @@
     "build": "ui5 build --clean-dest",
     "build:opt": "ui5 build self-contained --clean-dest --all",
     "start": "ui5 serve --port 8080 -o index.html",
+    "start-coverage": "ui5 serve --port 8080 --config ui5-coverage.yaml",
     "start:dist": "ui5 serve  --port 8080 -o index.html --config ui5-dist.yaml",
     "ts-typecheck": "tsc --noEmit",
     "lint": "eslint webapp",
-    "ui5lint": "ui5lint",
-    "karma": "karma start",
-    "karma-ci": "karma start karma-ci.conf.js",
-    "karma-ci-cov": "karma start karma-ci-cov.conf.js",
-    "test": "npm run lint && npm run karma-ci-cov"
+    "test-runner": "ui5-test-runner --port 8081 --url http://localhost:8080/test/testsuite.qunit.html",
+    "test-runner-coverage": "ui5-test-runner --port 8081 --url http://localhost:8080/test/testsuite.qunit.html --coverage -ccb 60 -ccf 100 -ccl 85 -ccs 85",
+    "test-ui5": "start-server-and-test start-coverage http://localhost:8080 test-runner-coverage",
```

Comment:
2 things :
* The `--port` option is useless unless you want to monitor the progress. It may be confusing for people looking at it
* `ui5-test-runner` now offers the `--start` option which allows you to eliminate `start-server-and-test` 😄

Reply:
Done, thanks!